test/format/t.c

```diff
@@ -183,52 +200,36 @@ main(int argc, char *argv[])
         }
     argv += __wt_optind;
 
-    /* Initialize the global RNG. */
-    __wt_random_init_seed(NULL, &g.rnd);
-
     /* Set up paths. */
     path_setup(home);
 
-    /* If it's a replay, use the home directory's CONFIG file. */
-    if (g.replay) {
+    /*
+     * If it's a replay or a reopen, use the already existing home directory's CONFIG file.
```

Comment:
Somewhere it would be good, maybe up where they're set in the args parsing, to comment about the difference between replay and reopen.

Reply:
Fixed in 848815c20.